src/views/EmotePage/EmotePage.vue

```diff
@@ -74,24 +70,33 @@
 				<div class="is-content-block">TODO</div>
 			</div>
 		</template>
-		<template v-else-if="!partial">Loading...</template>
+		<template v-else-if="loading">Loading...</template>
+		<template v-else>
+			<div class="emote-unknown">
+				<NotFoundPage></NotFoundPage>
```

Comment:
On components that have no children, it looks neater when doing the 
```
<NotFoundPage />
```
over the extended syntax